src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule72thesummaryfragment/InputIncorrectSummaryJavaDocCheck.java

```diff
@@ -85,32 +85,32 @@ void foo6() {}
       */
     InnerInputCorrectJavaDocParagraphCheck anon = new InnerInputCorrectJavaDocParagraphCheck() {
 
-        /**
-         * Some Javadoc.
-         */
-        public static final byte NUL = 0;
+            /**
```

Comment:
These changes look superflorous. You just indented code over and I don't see a reason for it.
Please undo.